xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -786,7 +786,7 @@ def main():
 
     idelayctrl_instances = place_constraints.get_used_instances("IDELAYCTRL")
 
-    assert len(idelayctrl_cmts) == len(
+    assert len(idelayctrl_cmts) <= len(
         idelayctrl_instances
```

Comment:
This will not operate properly, the previous assertion is correct.  In order to relax this contraint, the synthesis tool needs to emit duplicate IDELAYCTRL and tie their outputs together with a `AND<n>` LUT.  See TODO/FIXME above or below this code.